lib/java/ydoc-polyfill/src/test/java/org/enso/ydoc/polyfill/web/EventTargetTest.java

```diff
@@ -43,7 +43,15 @@ public void dispatchEvent() throws Exception {
         """
         var count = 0;
         var et = new EventTarget();
-        et.addEventListener('inc', () => count += 1);
+        var l = () => count += 1;
+        et.addEventListener('inc', l);
+        var arr = et.getEventListeners('inc');
+        if (arr.length != 1) {
+            throw 'Expecting one listener';
+        }
+        if (arr[0] != l) {
+            throw 'Expecting the one listener!';
+        }
```

Comment:
This test will work even with the original `Value[] getEventListeners()` implementation returning an array of values because we explicitly allow array access when creating a context: https://github.com/enso-org/enso/blob/7a0c8d2b8b9647ce347f7c47d848baacba424716/lib/java/ydoc-polyfill/src/main/java/org/enso/ydoc/polyfill/web/WebEnvironment.java#L13-L14

If we want to use `ProxyArray`s, please remove the explicit `allowArrayAccess(true)` access from the context to make the code consistent (I don't remember other places where arrays are returned but the appropriate tests should fail)

Reply:
- I see. So the diference is `.allowArrayAccess`!
- thanks for pointing that out!